occupancy_gridmap_server:
  #
  # Topics
  occupancyGridMapTopic: "/map"
  inflatedGridMapTopic: "/map_inflated"

  # Frame Ids
  mapFrame: "map"

  # Occupancy Map Image
  imagePath: "$(find occupancy_gridmap_server)/map/sample.png"
  gridResolution: 0.1 # [m/cell]
  freeThreshold: 0.3
  occupiedThreshold: 0.7

  # Occupancy Map Inflation
  inflationRadius: 0.3 # [m]

  # Map publish rate
  publishRate: 1.0 # [Hz]
